packages/web-components/src/drawer/drawer.styles.ts

```diff
@@ -0,0 +1,192 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/index.js';
+import {
+  colorBackgroundOverlay,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  colorTransparentStroke,
+  curveAccelerateMid,
+  curveDecelerateMid,
+  curveLinear,
+  durationGentle,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  shadow64,
+  strokeWidthThick,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+/** Drawer styles
+ * @public
+ */
+export const styles = css`
+  ${display('block')}
+
+  :host {
+    --dialog-backdrop: ${colorBackgroundOverlay};
+    overflow: hidden;
+  }
+
+  dialog {
+    position: fixed;
+    display: none;
+    z-index: var(--drawer-elevation, 1000);
```

Comment:
Do we need this if the position is fixed or this is a modal?

Reply:
we do not